src/forms/AdminEpochForm.tsx

```diff
@@ -1,32 +1,43 @@
+import { useState, useMemo } from 'react';
```

Comment:
move this file into `pages/HistoryPage` please

Reply:
can I rename it to EpochForm or that would lead to unwanted conflicts?

Reply:
done